setup.py

```diff
@@ -63,7 +63,7 @@
                       'tzlocal',
                       'certifi',
                       'packaging',
-                      'importlib-metadata',
+                      "importlib_metadata;python_version<'3.8'",
```

Comment:
The quoting can just be inverted to match the rest of the entries. For example:
```
'importlib_metadata;python_version<"3.8"'
```

Reply:
Done!